injitest/src/main/java/inji/pages/DetailedVcViewPage.java

```diff
@@ -130,7 +134,8 @@ public boolean isActivateButtonDisplayed() {
         return this.isElementDisplayed(activateButton);
     }
 
-    public PleaseConfirmPopupPage clickOnActivateButton() {
+    public PleaseConfirmPopupPage clickOnActivateButton(){
```

Comment:
clickOnActivateButtonAndroid()
clickOnActivateButtonIos()

and call them separately in tests. Need to google